scivision/catalog/data/datasources.json

```diff
@@ -256,6 +256,26 @@
       "format": "image",
       "name": "cell-cycle-scivision",
       "description": "Crops of H2B-FP stained nuclei in MDCK cells. "
+    },
+    {
+      "tasks": [
+        "segmentation"
+      ],
+      "labels_provided": false,
+      "domains": [
+        "fluorescent microscopy"
+      ],
+      "institution": [
+        "Broad Institute"
+      ],
+      "tags": [
+        "fluorescence microscopy ",
+        "bioimaging"
+      ],
+      "name": "Phenotypic Profiling",
+      "description": "Plate 20585 - Phenotypic profiling attempts to summarize multiparametric, feature-based analysis of cellular phenotypes of each sample so that similarities between profiles reflect similarities between samples. This image set provides a basis for testing image-based profiling methods wrt. to their ability to distinguish the effects of small molecules. Images The images are of U2OS cells treated with each of 1600 known bioactive compounds and labeled with six labels that characterize seven organelles (the .cell-painting. assay). This pilot experiment consists of 20 plates 5 of which are available. Each plate has 384 wells and each well has 9 fields of view for a total of 17,280 fields of view. Each field was imaged in five channels (detection wavelengths), and each channel is stored as a separate, grayscale image file, so there are 86,400 image files in 16-bit TIFF format.",
+      "url": "https://github.com/Tonks684/phenotypic_profiling/blob/main/data.yml",
+      "format": ".tif"
```

Comment:
Noting that we should figure out what to do with the 'format' field (in general) - #110, #474

Reply:
Yeah I wasnt sure if this meant image or given its based on computer vision ie images what type of image.